packages/common/client-utils/src/test/jest/gitHash.spec.ts

```diff
@@ -44,50 +48,57 @@ async function evaluateBrowserHash(
 	// so pull in the function as a string and eval it directly instead
 	// there are also issues around nested function calls when using page.exposeFunction, so
 	// do only the crypto.subtle part in page.evaluate and do the other half outside
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
 	const browserHashFn = HashBrowser.__get__("digestBuffer").toString();
-	const hashCharCodeString = await (page.evaluate(
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+	const hashCharCodeString = await page.evaluate(
```

Comment:
Similar to above; would adding `jest` to the tsconfig types help to remove some of the overrides?

Reply:
"jest" is already included in the test directory's tsconfig, so I don't believe that's the issue here. I don't remember why this one didn't get better typing. I'll double check.

Reply:
This was apparently just `evaluate` being typed to return `any`. Casting to a string resolved this particular issue, though I was also able to make a couple of other improvements inside the callback.

Good catch!